odmObject/odmobject-accelerometer.sdf.json

```diff
@@ -0,0 +1,60 @@
+{
+  "info": {
+    "title": "OMA LwM2M Accelerometer (Object ID 3313)",
+    "version": "20191002",
+    "copyright": "Copyright (c) 2018 IPSO.",
```

Comment:
also 2019?
(for all objects..)

Reply:
Good point. I simply copied this from the source file but indeed now it should contain also 2019.